src/test/java/core/basesyntax/dao/OperationDaoTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.model.Operation;
+import org.junit.Before;
+import org.junit.Test;
+
+public class OperationDaoTest {
+    private static Operation operation;
+    private static OperationDaoImpl operationDao;
+    private static final int SIZE_OPERATIONS = 4;
+
+    @Before
+    public void setUp() {
+        operation = new Operation("BALANCE", "b", Operation.ArithmeticOperation.ADD);
+        operationDao = new OperationDaoImpl();
+    }
+
+    @Test
+    public void amountOperations_Ok() {
+        assertEquals(operationDao.getListOperations().size(), SIZE_OPERATIONS);
```

Comment:
The first var is expected, and the second var is actual. Change order. FIx it in all places.